src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Media/ImageBrushTests/ImageBrush_WriteableBitmap.xaml.cs

```diff
@@ -18,7 +18,7 @@
 
 namespace UITests.Windows_UI_Xaml_Media.ImageBrushTests
 {
-	[Sample(typeof(ImageBrush), typeof(WriteableBitmap), IgnoreInSnapshotTests = true)]
+	[Sample("ImageBrushTests", nameof(WriteableBitmap), IgnoreInSnapshotTests = true)]
```

Comment:
"ImageBrushTests" doesn't exists:

![image](https://user-images.githubusercontent.com/8635919/89427234-06fad200-d709-11ea-9b77-b2e3c8045859.png)


```suggestion
	[Sample("ImageBrushTestControl", nameof(WriteableBitmap), IgnoreInSnapshotTests = true)]
```

But actually I would prefer to change all others to "ImageBrush"

Reply:
Actually this is invalid : the `nameof(WriteableBitmap)` is not resolved properly by the snapshot test generator : https://github.com/unoplatform/uno/blob/master/src/SamplesApp/SamplesApp.UITests.Generator/SnapShotTestGenerator.cs#L73

You must not use `nameof()` in the sample attributes.